src/views/ScreenSplitter.vue

```diff
@@ -19,13 +19,7 @@ de:
     :class="{'is-multipanel' : panels.length > 1}"
   )
     component.fill-panel(:is="panel.component" v-bind="panel.props" @navigate="onNavigate(i,$event)")
-    .control-buttons
```

Comment:
This is unrelated to fixing the stacked bar chart; is this supposed to be on another branch?